das-wg-charter.html

```diff
@@ -1557,27 +1589,38 @@ <h3 id="history">
                 Added Contact Picker API, Idle Detection API, and Compute Pressure to the group's tentative deliverables.  Changed the Fold Angle Sensor API from a tentative to a normative deliverable, renaming it as the Device Posture API.
               </td>
             </tr>
+            <tr>
+              <th>
+                <a href="">Rechartered</a>
+              </th>
+              <td>
+                <i class="todo">[YYYY-MM-DD]</i>
+              </td>
+              <td>
+                <i class="todo">[YYYY-MM-DD]</i>
+              </td>
+              <td>
+                Marked Screen Wake Lock API, DeviceOrientation Event specification, Contact Picker API, and Geolocation API as
```

Comment:
Contact Picker API was published as a joint deliverables already under the 2022-12-01 charter even if it was not explicitly marked as such. I recall we had an agreement on it with WebApps WG but somehow that text did not make it to the charter back then. It is probably worth clarifying that just in case someone wonders why Contact Pickier API was published as a joint deliverable in 20 December 2022.

Reply:
@anssiko how about text as 9815550?